src/+otp/+sanzserna/SanzSernaProblem.m

```diff
@@ -21,6 +26,11 @@ function onSettingsChanged(obj)
             obj.Rhs = otp.Rhs(@(t, y) otp.sanzserna.f(t, y, D, x), ...
                 otp.Rhs.FieldNames.Jacobian, @(t, y) otp.sanzserna.jac(t, y, D, x), ...
                 otp.Rhs.FieldNames.JacobianVectorProduct, @(t, y, v) otp.sanzserna.jvp(t, y, v, D, x));
+            
+            obj.RhsLinear = otp.Rhs(@(t, y) otp.sanzserna.flinear(t, y, D, x), ...
+                otp.Rhs.FieldNames.Jacobian, @(t, y) otp.sanzserna.jaclinear(t, y, D, x));
+            obj.RhsForcing = otp.Rhs(@(t, y) otp.sanzserna.fforcing(t, y, D, x), ...
+                otp.Rhs.FieldNames.Jacobian, @(t, y) otp.sanzserna.jacforcing(t, y, D, x));
```

Comment:
as we have jacobian vector products on the Rhs, we might as well do it on these as well.

Reply:
I can't imagine a scenario where we would need it and don't think it's worth adding.